src/ai/backend/agent/stats.py

```diff
@@ -524,12 +525,17 @@ async def collect_per_container_process_stat(
             results = await asyncio.gather(*_tasks, return_exceptions=True)
             updated_cids: Set[ContainerId] = set()
             for result in results:
-                if isinstance(result, Exception):
-                    log.error(
-                        "collect_per_container_process_stat(): gather_process_measures() error",
-                        exc_info=result,
-                    )
-                    continue
+                match result:
+                    case psutil.NoSuchProcess():
+                        log.debug(str(result))
+                        continue
+                    case Exception():
+                        log.error(
+                            "collect_per_container_process_stat(): gather_process_measures() error",
+                            exc_info=result,
+                        )
```

Comment:
```suggestion
                    case Exception():
                        log.error(
                            "collect_per_container_process_stat(): gather_process_measures() error",
                            exc_info=result,
                        )
                        continue
```
oops! miss the `continue` here